features/net/FEATURE_IPV4/TESTS/mbedmicro-net/udp_echo_client/main.cpp

```diff
@@ -66,6 +66,8 @@ int main() {
             result = false;
```

Comment:
I don't think so. 
Result is set to true at each loop, so before break, we need to set it back to false